backend/crates/kartoffel/src/panic.rs

```diff
@@ -1,10 +1,18 @@
+use core::fmt::Write;
 use core::panic::PanicInfo;
 
+use crate::SerialOutput;
+
 #[allow(dead_code)]
 #[allow(clippy::empty_loop)]
 #[cfg_attr(target_arch = "riscv64", panic_handler)]
-fn panic(_: &PanicInfo) -> ! {
+fn panic(i: &PanicInfo) -> ! {
+    let mut output = SerialOutput;
+
+    let _ = write!(&mut output, "\n- KERNEL PANIC -\n");
```

Comment:
Usually I'm into an extra bit of humor, but I worry that people might get the wrong impression here 😅 -- would you mind dropping this message?